releasing/README.md

```diff
@@ -4,44 +4,202 @@
 [`cloud-build-local`]: https://github.com/GoogleCloudPlatform/cloud-build-local
 [Google Cloud Build]: https://cloud.google.com/cloud-build
 [semver]: https://semver.org
+[Go modules]: https://github.com/golang/go/wiki/Modules
 
-Scripts and configuration files for publishing a
-`kustomize` release on the [release page].
+This document describes how to perform a [semver] release
+of one of the [Go modules] in this repository.
 
-## Build a release locally
+These modules release independently.
 
-Install [`cloud-build-local`], then run
+| Module Description | Module Prefix | Tag | Branch Name | 
+| ---         | ----        | --- | --- |
+| the kustomize executable | `sigs.k8s.io/kustomize/kustomize` | _kustomize/v{major}.{minor}.{patch}_ |_release-kustomize-v{major}.{minor}_ |
+| the kustomize Go API | `sigs.k8s.io/kustomize`   | _v{major}.{minor}.{patch}_ |_release-api-v{major}.{minor}_ |
+| the pluginator executable | `sigs.k8s.io/kustomize/pluginator` | _pluginator/v{major}.{minor}.{patch}_ | _release-pluginator-v{major}.{minor}_ |
+
+
+## Module summaries
+
+### sigs.k8s.io/kustomize/kustomize
+
+The `kustomize` program, a CLI for performing
+k8s-aware structured edits to YAML in k8s resource
+files.
+
+#### Packages
+
+There's only one package in this module.  It's called `main`,
+and it holds the _kustomize_ executable.
+
+
+#### Release artifacts
+
+Executable files appear for various operating systems
+on the [release page].
+
+See the [installation instructions](../INSTALL.md)
+to perform an install.
+
+
+### sigs.k8s.io/kustomize
+
+This is the kustomize Go api.
+
+#### Packages
+
+The packages in this module are used by API clients,
+like the `kustomize` program itself, and programs
+in other repositories, e.g. `kubectl`.
+
+They include methods to read, edit and emit
+modified YAML.
+
+Go consumers of this API will have a `go.mod` file
+_requiring_ this module at a particular tag, e.g.
 
 ```
-./releasing/localbuild.sh
+require sigs.k8s.io/kustomize/v4 v4.0.1
 ```
 
-## Do a real (cloud) release
+#### Release artifacts
+
+Source code tar balls and zip files (no executables)
+appear on the [release page].
+
+### sigs.k8s.io/kustomize/pluginator
```

Comment:
Can you also explain the policy of when to release a new version of pluginator and when to update the pluginator in the api module?

Reply:
done